core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/utils/ReplicatedTableProviderImpl.java

```diff
@@ -52,14 +61,66 @@ public boolean accept(String keyspace)
             return false;
         }
 
-        Set<TokenRange> tokenRanges = myMetadata.getTokenRanges(keyspace, myLocalhost);
+        KeyspaceMetadata keyspaceMetadata = myMetadata.getKeyspace(keyspace);
 
-        if (tokenRanges.isEmpty())
+        if (keyspaceMetadata != null)
         {
+            Map<String, String> replication = keyspaceMetadata.getReplication();
+            String replicationClass = replication.get(STRATEGY_CLASS);
+
+            switch(replicationClass)
+            {
+                case SIMPLE_STRATEGY:
+                    return validateSimpleStrategy(replication);
+                case NETWORK_TOPOLOGY_STRATEGY:
+                    return validateNetworkTopologyStrategy(keyspace, replication);
+                default:
+                    LOG.warn("Replication strategy of type {} is not supported", replicationClass);
```

Comment:
Can I get rid of this warning message if I've created my own custom strategy? If not, should it be at info-level?

Reply:
Not as of now, if you don't provide your own ReplicatedTableProvider.. But I think it make sense to have this as warning still since no repair would be scheduled for this table with the default providers.

Reply:
Fair enough